src/playermat/Playermat.ttslua

```diff
@@ -341,19 +348,14 @@ function doUpkeep(_, clickedByColor, isRightClick)
   local forcedLearning = false
   local rot = self.getRotation()
   for _, obj in ipairs(searchAroundSelf()) do
-    if obj.hasTag("Temporary") == true then
+    if obj.hasTag("Temporary") then
       discardListOfObjects({ obj })
-    elseif obj.hasTag("UniversalToken") == true and obj.is_face_down then
+    elseif obj.hasTag("UniversalToken") and obj.is_face_down then
       obj.flip()
-    elseif obj.type == "Card" then      
-      if obj.hasTag("DoInUpkeep") then
-        obj.call("doInUpkeep")
-      end
-      -- do not rotate, replenish, etc. on cards in investigator card area
-      if inArea(self.positionToLocal(obj.getPosition()), INVESTIGATOR_AREA) then
-        break
-      end
-      
+    elseif obj.hasTag("DoInUpkeep") and obj.is_face_down then
```

Comment:
I kept this from being a separate elseif in case of a future card that has the DoInUpkeep tag and still has uses to replenish, to be rotated, etc.

Reply:
That's fair, I'll update it later!